astropy/io/ascii/tests/test_html_unicode.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+# TEST_UNICODE_LITERALS
+
+import numpy as np
+from ....table import Table
+from ....tests.helper import pytest
+
+# Check to see if the BeautifulSoup dependency is present.
+try:
+    import bs4
+    HAS_BEAUTIFUL_SOUP = True
+except ImportError:
+    HAS_BEAUTIFUL_SOUP = False
+
+@pytest.mark.skipif('not HAS_BEAUTIFUL_SOUP')
+def test_read_html_unicode():
+    """
+    Test reading a table with unicode values
+    """
+    # If we don't have unicode literals then return
+    if isinstance('', bytes):
```

Comment:
To avoid this and using `TEST_UNICODE_LITERALS`, the strings below could simply be defined with the `u'...'` syntax, or maybe I miss something ?